view/template/content/faq.php

```diff
@@ -44,4 +44,10 @@
       <?php endforeach ?>
     </div>
   </section>
+
+  <section>
+    <div class="inner-wrap">
+      <p>Do you have further questions or want to submit a question to our FAQ? <a href=" mailto:growth@lbry.com?subject=Question%20for%20the%20FAQ" rel="noopener noreferrer" target="_blank" title="">Send us an email</a>!</p>
```

Comment:
Is this the correct email address to use?

Reply:
@kauffj ^

Reply:
@tzarebczan any thoughts or preferences on how to collect these? We could link a Google form or just invite to email.